src/server/hook_func.c

```diff
@@ -4619,17 +4619,13 @@ int server_process_hooks(int rq_type, char *rq_user, char *rq_host, hook *phook,
 			}
 
 			if (job_modified != 1) {
-					new_conv_str =
+				new_conv_str =
 					pbs_python_event_job_getval_hookset(
-					ATTR_create_resv_from_job, NULL, 0, NULL, 0);
+						ATTR_create_resv_from_job, NULL, 0, NULL, 0);
 
-				if (new_conv_str != NULL) {
-					snprintf(log_buffer, sizeof(log_buffer),
-						"Found job '%s' attribute flagged to be set",
-						ATTR_create_resv_from_job);
-					log_event(PBSEVENT_DEBUG2, PBS_EVENTCLASS_HOOK, LOG_ERR, phook->hook_name, log_buffer);
-
-				}
+				if (new_conv_str != NULL)
+					log_eventf(PBSEVENT_DEBUG2, PBS_EVENTCLASS_HOOK, LOG_ERR, phook->hook_name,
+						"Found job "ATTR_create_resv_from_job" attribute flagged to be set");
```

Comment:
Since the %s was taken out, you can just use log_event() now.  Also, put spaces after the " and before the next opening " around the macro.  Right now it looks like it is quoted and not the other strings.